server/cache_utils.py

```
@@ -0,0 +1,1093 @@
+    # 3) Fallback by file types
+    snapshots_dir = Path(model_path) / "snapshots"
+    if not snapshots_dir.exists():
+        return "Unknown"
+    has_gguf = any(snapshots_dir.glob("*/*.gguf"))
+    has_safetensors = any(snapshots_dir.glob("*/*.safetensors"))
+    has_pytorch_bin = any(snapshots_dir.glob("*/pytorch_model.bin"))
+    has_config = any(snapshots_dir.glob("*/*.json"))
+    total_size = get_model_size(Path(model_path))
+    try:
+        size_mb = float(total_size.replace(" GB", "000").replace(" MB", "").replace(" KB", "0").replace(" ", ""))
+    except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)